### PR TITLE
Uninstall landscaper correctly

### PR DESCRIPTION
**Pull request by achimweigel**
**What this PR does / why we need it**:

Uninstall complete landscaper including deployer registrations and namespace

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```feature user
Uninstall complete landscaper including deployer registrations and namespace.
```

### CODE REVIEW

File: cmd/quickstart/uninstall.go [hunk 1]
```diff
@@ -105,15 +113,69 @@ func (o *uninstallOptions) uninstallOCIRegistry(ctx context.Context, k8sClient c
...
+		if err := k8sClient.Delete(ctx, &registration); err != nil {
+			return err
+		}
+	}
+
+	if err := waitForRegistrationsRemoved(ctx, k8sClient); err != nil {
+		return err
+	}
+
+	fmt.Println("helm delete")
```
Comment (jschicktanz) [inline cmd/quickstart/uninstall.go:133]: Think this is not needed as the "helm delete" command will already be logged to console in util. ExecCommandBlocking() from the next line

**Review by robertgraeff — COMMENTED** [review]
Should we increase the timeout when waiting for the deletion of the deployer registrations. It worked locally, but the integration test fails.

File: cmd/quickstart/uninstall.go (hunk 1 above)
Comment (achimweigel) [reply cmd/quickstart/uninstall.go]: I have removed the line

**Comment by achimweigel** [comment]
> Should we increase the timeout when waiting for the deletion of the deployer registrations. It worked locally, but the integration test fails.

The tests are failing due to some other stuff. Nevertheless I have increased the timeout.

**Review by jschicktanz — APPROVED** [review]

**Pull request merged**